build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -425,6 +429,13 @@ antreaProxy:
   # enabled. This avoids race conditions between kube-proxy and Antrea Proxy, with both trying to
   # bind to the same address, when proxyAll is enabled while kube-proxy has not been removed.
   disableServiceHealthCheckServer: {{ .disableServiceHealthCheckServer }}
+  # The value of the IP address and the port on which AntreaProxy health server listens when proxyAll is enabled. This
+  # server is functionally equivalent to the one of kube-proxy. If it is not specified, it will be automatically set
+  # to "0.0.0.0:10352" for IPv4-only clusters, or "[::]:10352" for IPv6-only and dual-stack clusters. Note that
+  # "[::]:10352" may act as either IPv6-only or dual-stack depending on the system setting of IPV6_V6ONLY
+  # (sysctl: net.ipv6.bindv6only).
+  serviceHealthCheckServerBindAddress: {{ .serviceHealthCheckServerBindAddress | quote }}
```

Comment:
Personally I think we may want to keep using the same default (`10256`), otherwise it could break code that hardcodes this value? It's good to have a way to change this value in the config though.

What do you think?

Reply:
I'm fine with the default `10256`. Do we need to still add the port to `docs/network-requirements.md` and `apis/port.go`?